### PR TITLE
add pdf table parsing to the html table sample

### PR DESCRIPTION
**Pull request by CMPeng**
Per discussion in scrum, adding the part of "PDF table -> pdf map" sample to the existing "html table -> portal item" sample.

I have been seeing errors being thrown at the original "html table -> portal item" sample, so some changes are being made there as well.

### CODE REVIEW

File: samples/05_content_publishers/html_table_to_pandas_data_frame_to_portal_item.ipynb [hunk 1]
```diff
@@ -4,7 +4,24 @@
    "cell_type": "markdown",
```
Comment (Yongyao) [inline samples/05_content_publishers/html_table_to_pandas_data_frame_to_portal_item.ipynb]: <p>change to online_profile?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (Yongyao) [inline samples/05_content_publishers/html_table_to_pandas_data_frame_to_portal_item.ipynb]: <p>"If such data contained location information", <code>contained</code> to <code>contains</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

**Comment by Yongyao** [comment]
@CMPeng Well-written sample. Just made some minor comments through reviewNB

**Comment by CMPeng** [comment]
thanks @Yongyao 
Changes are made according to revision suggestions.

**Comment by CMPeng** [comment]
@AtmaMani Please check the latest commit, in which the "html table" and "pdf table" are split into two notebooks. The "html table" is included here because some lines in the original file is not applicable now. Thanks!

File: samples/05_content_publishers/pdf_table_to_pdf_map.ipynb [hunk 2]
```diff
@@ -0,0 +1,1459 @@
+{
```
Comment (AtmaMani) [inline samples/05_content_publishers/pdf_table_to_pdf_map.ipynb]: <ol><li>no apostrophe in "panda's data frame". -&gt; Pandas <code>DataFrame</code></li><li>Missing space in "machinebefore" words</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (AtmaMani) [inline samples/05_content_publishers/pdf_table_to_pdf_map.ipynb]: <p>Could you check if CAD stands for "Computer Aided Dispatch (CAD) system"? I thought it stood for Computer Aided Drawing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (AtmaMani) [inline samples/05_content_publishers/pdf_table_to_pdf_map.ipynb]: <p>This is very useful and readers would like to reuse this. Could you add some comments explaining how new rows are added? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

**Comment by AtmaMani** [comment]
@CMPeng this is a nice article, I have requested for a couple of edits

**Comment by CMPeng** [comment]
Thanks @AtmaMani 
Your requests of changes have been addressed in the latest commit.
PLUS. I have removed the short-hand form of CAD from the texts since CAD means computer aided design, and also computer aided dispatch, and can cause ambiguity.

**Pull request merged**